crates/octez/src/async/client.rs

```diff
@@ -25,8 +23,7 @@ pub struct OctezClientBuilder {
     binary_path: Option<PathBuf>,
     // if None, use temp directory
     base_dir: Option<PathBuf>,
-    // if None, use localhost:8732 (the default endpoint for octez-node)
-    endpoint: Option<Endpoint>,
+    octez_node_endpoint: Option<Endpoint>,
```

Comment:
I suggest to make this a required field and add fn with_node_config

Reply:
i think we can have two constructions: 
```
fn new(octez_node_endpoint) -> Self;
fn with_node_config(config: OctezNodeConfig);
```